Tests/demistomock/demistomock.py

```diff
@@ -1044,3 +1044,49 @@ def getModules():
 
 def get_incidents():
     return {}
+
+
+def internalHttpRequest(method, uri, body=None):
+    """Run an internal HTTP request to the XSOAR server. The request runs with the permissions of the 
+    executing user, when a command is being executed manually (such as via the warroom or when browsing a widget).
+    When run via a playbook, will run with a readonly user with limited permissions isolated to the current incident only.
+    Available for both Integrations and Scripts starting from Server version 6.1.
```

Comment:
dont think need to mention it is available for both, as for other functions we note if its for only integrations/scripts, and if its available in both we just dont comment that

Reply:
Worth mentioning as this is a special one.